lib/auth/grpcserver.go

```diff
@@ -2209,6 +2209,8 @@ func (g *GRPCServer) GetOIDCConnector(ctx context.Context, req *types.ResourceWi
 	if !ok {
 		return nil, trace.Errorf("encountered unexpected OIDC connector type %T", oc)
 	}
+	// DELETE IN 11.0.0
+	connector.CheckAndSetRedirectURL()
```

Comment:
Can we somehow ensure these fixups happen in a single place? Right now it is hard to ensure correctness - very easy to miss some code path.